docs/topics/pagination.txt

```diff
@@ -161,6 +161,17 @@ Optional arguments
     Whether or not the first page is allowed to be empty.  If ``False`` and
     ``object_list`` is  empty, then an ``EmptyPage`` error will be raised.
 
+``request``
+    An ``HttpRequest`` object, used for creating pagination query strings.
+
+    See :meth:`~Page.next_page_querystring`.
+
+``page_kwarg``
```

Comment:
you are commenting on the "page_kwarg" argument, right?

i chose to use the same name here as in "MultipleObjectMixin":
https://github.com/django/django/blob/0ed7d155635da9f79d4dd67e4889087d3673c6da/django/views/generic/list.py#L12

... that way the built in pagination follows the convention set by the class based view.